packages/safe-ds-eda/src/components/profiling/ProfilingInfo.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
+    import type { Profiling, ProfilingDetail } from '../../../types/state';
+
+    export let profiling: Profiling;
+    export let imageWidth: number = 200;
+
+    const getProfilingItemColor = function (profilingItem: ProfilingDetail) {
+        if (profilingItem.interpretation === 'good') {
+            return 'var(--primary-color)';
+        } else if (profilingItem.interpretation === 'error') {
+            return 'var(--error-color)';
+        } else if (profilingItem.interpretation === 'warn') {
+            return 'var(--warn-color)';
+        } else if (profilingItem.interpretation === 'bold') {
+            return 'var(--font-dark)';
+        } else if (profilingItem.interpretation === 'default') {
+            return 'var(--font-light)';
+        } else if (profilingItem.interpretation === 'category') {
+            return 'var(--font-light)';
+        } else {
+            return 'var(--font-light)';
+        }
+    };
+</script>
+
+<div class="wrapper">
+    <div class="profilingItemsTop">
+        {#each profiling.top as profilingTopItem}
+            {#if profilingTopItem.type === 'text'}
+                <div class="profilingItem">
+                    <span style="color: {getProfilingItemColor(profilingTopItem)};">{profilingTopItem.value}</span>
```

Comment:
Where possible, keep styling out of JS. Here, you can use `profilingItem.interpretation` in a class attribute and set the colors via CSS.